waf/azurermStorageAccount.go

```diff
@@ -0,0 +1,12 @@
+package waf
+
+import "github.com/Azure/tflint-ruleset-avm/attrvalue"
+
+// AzurermStorageAccountAccountReplicationType checks whether the account_replication_type is set to ZRS.
+var AzurermStorageAccountAccountReplicationType = func() *attrvalue.StringRule {
+	return attrvalue.NewStringRule(
+		"azurerm_storage_account",
+		"account_replication_type",
+		[]string{"ZRS"},
```

Comment:
Should we also include GA-ZRS?

Reply:
I think this depends on what we want AVM to be.

We state that we should be HA by default but do we state DR?

Remember this is just the default value provided by the module.

Reply:
yeah thats fair, I was thinking that an OR for `ZRS` or `GZRS` would satisfy the test as one is just an enhancement on the other